docs/attributes-registry/tls.md

```diff
@@ -10,37 +10,37 @@
-| `tls.server.not_after`         | string   | Date/Time indicating when server certificate is no longer considered valid.                                                                                                                                                         | `2021-01-01T00:00:00.000Z`                                                                  | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `tls.server.not_before`        | string   | Date/Time indicating when server certificate is first considered valid.                                                                                                                                                             | `1970-01-01T00:00:00.000Z`                                                                  | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `tls.server.subject`           | string   | Distinguished name of subject of the x.509 certificate presented by the server.                                                                                                                                                     | `CN=myserver, OU=Documentation Team, DC=example, DC=com`                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Attribute                      | Type     | Description                                                                                                                                                                                                                         | Examples                                                                                              | Stability                                                        |
+| ------------------------------ | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `tls.cipher`                   | string   | String indicating the [cipher](https://datatracker.ietf.org/doc/html/rfc5246#appendix-A.5) used during the current connection. [1]                                                                                                  | `TLS_RSA_WITH_3DES_EDE_CBC_SHA`; `TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256`                              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
I wonder why these spaces come from, makes the diff large 🤔

Reply:
As above... the table padding was added in the previous release. See PR #917